packages/core/status/core-status-server-internal/src/routes/status_preboot.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { IRouter } from '@kbn/core-http-server';
+import { ServiceStatusLevels } from '@kbn/core-status-common';
+
+export const registerPrebootStatusRoute = ({ router }: { router: IRouter }) => {
+  router.get(
+    {
+      path: '/api/status',
```

Comment:
Second root change of the PR, register a `/api/status` route on the preboot server, always returning `503 { status: "unavailable" }`